crates/server/src/admin/service.rs

```diff
@@ -150,21 +159,12 @@ pub(crate) fn setup(
         )
         .route(
             "/contexts/:context_id/proposals",
-            get(get_proposals_handler),
+            post(get_proposals_handler),
```

Comment:
is this right?

Reply:
Can't pass it properly from frontend with get as it takes in body

Reply:
Ah, okay..